### PR TITLE
Add Hindi README and Update Main README Reference 

### PR DESCRIPTION
**Pull request by NamanVer02**
## Add Hindi README and Update Main README Reference  

**Describe the change**  
This pull request introduces a new README file in Hindi (`README.hi.md`) to make the project more accessible to Hindi-speaking users. Additionally, the main `README.md` has been updated to include a reference to the new Hindi README.

**Additional context**  
This change aims to broaden the reach of the repository and support users who are more comfortable learning Python in Hindi. If you have any questions or need further adjustments, feel free to ask!

### CODE REVIEW

**Review by huangsam — CHANGES_REQUESTED** [review]
Please add this new linkage in all of the README files

File: README.md [hunk 1]
```diff
@@ -16,7 +16,8 @@ print("Ultimate Python study guide")
 [한국어](README.ko.md) |
 [繁体中文](README.zh_tw.md) |
 [Español](README.es.md) |
-[Deutsch](README.de.md)
+[Deutsch](README.de.md) |
+[हिन्दी](README.hi.md)
```
Comment (huangsam) [inline README.md:20]: Put this in the following README files:
- README.ko.md
- README.zh_tw.md
- README.es.md
- README.de.md
If there are any others I missed, please put them in there as well.

**Review by huangsam — CHANGES_REQUESTED** [review]
Updated review with a comment to resolve

File: README.md (hunk 1 above)
Comment (NamanVer02) [reply README.md:20]: Sure, Ill update all the readme files with the reference.

**Review by huangsam — APPROVED** [review]

**Comment by huangsam** [comment]
Merged 🚀 - if you end up finding things you'd like to edit, feel free to submit another PR. Happy Hacktoberfest!

**Pull request merged**